### ISSUE TITLE
loss stuck at ~.69

### ISSUE THREAD

**Issue by gwohl** [BVLC/caffe#826]
I am training a network for binary classification. I am finetuningEvery time I try to train, at around the end of the first epoch, the loss jumps from ~.09 to ~.69 and then stays at ~.69. I am not sure what is causing this. I have tried various learning rates (.01, .001, .0001, .00001) and it happens every time. I have also tried to keep track of the number of positive and negative examples in each batch and on the batch where this happens and the batch is around 20% positive and 80% negative. 

My batch size is 256.

Any Ideas as to why this is happening?
